index.js

```diff
@@ -181,3 +185,55 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost('ideatown.dev', function(cookies) {
+    let csrf;
+    let cookieStr = '';
+    cookies.forEach(function(c) {
+      cookieStr += c.name + '=' + c.value + ' ';
+      if (c.name === 'csrftoken') csrf = c.value;
+    });
+
+    if (csrf) {
```

Comment:
I think we should always have a `csrftoken` cookie set. Have you encountered cases where it isn't set?